docs/Team/index.md

```diff
@@ -36,10 +48,19 @@ All developers will be able to perform code reviews and give constructive feedba
 The rest of these are more what you call [guidelines, not actual rules](https://youtu.be/jl0hMfqNQ-g).
 
 ## Advancement
-You've heard it said that you should dress for the job you want, well similarly, if you want to be a Senior or a Lead developer, you should already be demonstrating those traits. In other words if you ask for a promotion, it shouldn't cause your manager to hesitate.
+You've heard it said that you should dress for the job you want, well similarly, if you want to be a Senior Developer or a Manager, you should already be demonstrating the traits respective for those roles. In other words if you ask for a promotion, it shouldn't cause your manager to hesitate.
```

Comment:
First sentence: '...demonstrating the traits _required_ for those roles.' Replace 'respective' with 'required'

Reply:
How about "expected"?

Reply:
`job you want, well similarly` - comma should be a semicolon

Reply:
@swilliams 'expected' is fine